### ISSUE TITLE
Home office

### ISSUE THREAD

**Issue by sapegin** [survivejs/ama#11]
Could you show a photo of your workspace? What hardware do you use, etc.?

**Comment by bebraw** [comment]
Here we go:

![office](https://cloud.githubusercontent.com/assets/166921/21484415/b8c35e04-cb9a-11e6-8c79-a9b7c44f19df.jpeg)

My setup is simple and portable. I can literally carry most of my equipment with me. Quick specs below:

* Laptop - 15" MBP (Late 2016) - Top spec apart from HD as I settled for 512 GB HD. I bought an external HD (1 TB) for backups/photos and I push the most important data to the cloud. To improve ergonomics I have the laptop on a slight angle (Easel laptop stand).
* Tablet - iPad Air 2 (64 GB, LTE) - I use the iPad as my second display occasionally through Duet. It works quite well for running tests and previewing work. I also use it as my primary connection as LTE is so fast over here (faster than ADSL).
* Headphones - Bose QC-15 - I have used these headphones for something like four years at least. They were expensive, but a good investment still. I had to replace the pads recently. I bought Sennheiser Momentum 2's (wireless) as a replacement, but they are in warranty at the moment (extra noise at the right speaker). Momentum 2's have pretty good sound out of the box and I love the fact that they are wireless.
* Mic - Blue Yeti - I bought it as it's apparently pretty good. I just need to remember to move it to the table as otherwise it will echo in the box.
* Light - Some cheapo LED light as it's dark in the winter.
* Batteries - Sanyo Eneloops - I maintain Sanyo Eneloop setup for QC-15 and other devices that might require batteries. The initial investment is tens of euros, but it pays off pretty quickly as you don't have to worry about batteries anymore.
* Camera - Fujifilm X100S - I bought this one used based on a recommendation of a photographer friend. It's the perfect travel camera for me and if I need some custom stock photos, I can easily take them myself. The objective is fixed, but high quality.

I have an old MBA laying around as a spare laptop in case the primary one breaks. There's also an old Wacom Intuos in case I need to draw something.

I guess that's it. Maybe 27-30" high resolution display would be sweet, but apart from that it's hard to complain.
